packages/ffe-grid-react/src/GridRow.tsx

```diff
@@ -18,14 +25,38 @@ export type GridRowProps<As extends ElementType = any> =
         background?: BackgroundColor;
         /** Supported dark background colors */
         backgroundDark?: BackgroundColorDark;
+        /** Size modifiers for small screen sizes */
+        sm?: SizeModifier;
+        /** Size modifiers for medium screen sizes */
+        md?: SizeModifier;
+        /** Size modifiers for large screen sizes */
+        lg?: SizeModifier;
     };
 
+const sizeClasses = (size: 'sm' | 'md' | 'lg', def?: SizeModifier) => {
+    if (def === undefined) {
+        return null;
+    } else if (typeof def === 'number' || typeof def === 'string') {
```

Comment:
```suggestion
    } else if (typeof def === 'string') {
```
Kan ikke vare number